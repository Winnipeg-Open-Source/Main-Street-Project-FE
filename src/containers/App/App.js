import React from 'react';
import Router from 'containers/Router';

function App () {
    return <Router />;
}

App.displayName = 'App';

export default App;
