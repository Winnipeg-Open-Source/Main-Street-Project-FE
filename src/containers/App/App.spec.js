import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
    it('renders without crashing', () => {
        const { asFragment } = render(<App />);

        expect(asFragment()).toMatchSnapshot();
    });
});
