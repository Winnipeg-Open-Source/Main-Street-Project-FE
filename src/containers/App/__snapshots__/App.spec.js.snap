// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pbJYR hNzbLI sc-pkfPC fQMEfn"
  >
    <div
      class="sc-AxjAm sc-AxiKw sc-qYUWV jpqZzA"
      color="primary"
    >
      <svg
        aria-hidden="true"
        class="sc-Axmtr dFmDHs sc-pAZqv sc-pBlxj eigskg"
        data-testid="menu-icon"
        fill="currentcolor"
        focusable="false"
        height="24"
        role="img"
        tabindex="-1"
        title="Menu"
        viewBox="0 0 24 24"
        width="24"
      >
        <title>
          Menu
        </title>
        <path
          d="M3 13h18v-2H3v2zm0 5h18v-2H3v2zM3 6v2h18V6H3z"
        />
      </svg>
      Main Street Project Inventory System
    </div>
    <div
      class="sc-AxjAm sc-AxiKw kUKRqP"
      style="height: 100%;"
    >
      <div
        class="sc-AxjAm sc-AxiKw sc-psedN hRWWYQ"
        color="tertiary.dark"
        width="56"
      >
        <a
          class="sc-pjGMk WuYUh"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-fzoCUK gcKcJr"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M10 20.5v-6h4v6h5v-8h3l-10-9-10 9h3v8h5z"
            />
          </svg>
          Home
        </a>
        <a
          class="sc-pjGMk fqvTnB"
          href="/donors"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-pIjat gTixYT"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M12 12c2.2 0 4-1.8 4-4s-1.8-4-4-4-4 1.8-4 4 1.8 4 4 4zm0 2c-2.7 0-8 1.3-8 4v2h16v-2c0-2.7-5.3-4-8-4z"
            />
          </svg>
          Donors
        </a>
        <a
          class="sc-pjGMk fqvTnB"
          href="/donations"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-pjSSY laaBPC"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M21.4 11.6l-9-9c-.3-.4-.8-.6-1.4-.6H4c-1.1 0-2 .9-2 2v7c0 .6.2 1.1.6 1.4l9 9c.4.4.9.6 1.4.6s1.1-.2 1.4-.6l7-7c.4-.4.6-.9.6-1.4s-.2-1.1-.6-1.4zM5.5 7C4.7 7 4 6.3 4 5.5S4.7 4 5.5 4 7 4.7 7 5.5 6.3 7 5.5 7zm11.8 8.3L13 19.5l-4.3-4.3c-.4-.4-.7-1-.7-1.7 0-1.4 1.1-2.5 2.5-2.5.7 0 1.3.3 1.8.7l.7.7.7-.7c.5-.4 1.1-.7 1.8-.7 1.4 0 2.5 1.1 2.5 2.5 0 .7-.3 1.3-.7 1.8z"
            />
          </svg>
          Donations
        </a>
        <a
          class="sc-pjGMk fqvTnB"
          href="/items"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-fzqOul fQbLGO"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M22 16V4c0-1.1-.9-2-2-2H8c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2zm-11-4l2 2.7 3-3.7 4 5H8l3-4zM2 6v14c0 1.1.9 2 2 2h14v-2H4V6H2z"
            />
          </svg>
          Items
        </a>
        <a
          class="sc-pjGMk fqvTnB"
          href="/locations"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-qYiqT duDEIF"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M20.5 3h-.2L15 5.1 9 3 3.4 4.9c-.2.1-.4.3-.4.5v15.1c0 .3.2.5.5.5h.2L9 18.9l6 2.1 5.6-1.9c.2-.1.4-.2.4-.5V3.5c0-.3-.2-.5-.5-.5zM15 19l-6-2.1V5l6 2.1V19z"
            />
          </svg>
          Locations
        </a>
        <a
          class="sc-pjGMk fqvTnB"
          href="/requisitions"
        >
          <svg
            aria-hidden="true"
            class="sc-Axmtr cTdHPN sc-fznLPX llMHGf"
            fill="currentcolor"
            focusable="false"
            height="24"
            role="img"
            tabindex="-1"
            viewBox="0 0 24 24"
            width="24"
          >
            <path
              d="M14 2H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V8l-6-6zm2 16H8v-2h8v2zm0-4H8v-2h8v2zm-3-5V3.5L18.5 9H13z"
            />
          </svg>
          Requisitions
        </a>
      </div>
      <div
        class="sc-AxjAm sc-AxiKw sc-qQWDO cdkSxU"
        color="primary.light"
      >
        <div
          class="sc-AxjAm sc-AxiKw jthnRD"
          color="primary.light"
          width="1"
        >
          <a
            class="sc-pjGMk fqvTnB"
            href="/donations/new"
          >
            <button
              class="sc-oUOMp dmWTjG"
              color="primary"
              width="1"
            >
              New Donation
            </button>
          </a>
          <a
            class="sc-pjGMk fqvTnB"
            href="/requisitions/new"
          >
            <button
              class="sc-oUOMp dUIyXc"
              color="primary"
              width="1"
            >
              New Requisition
            </button>
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
